.github/workflows/create-release-main.yml

```diff
@@ -0,0 +1,71 @@
+name: Create new preview release (main)
+
+on:
+  # run when pushed to main is published,
+  # which creates a new tag
+  push:
+    branches:
+      - main
+
+env:
+  ARTIFACT_VERSION: experimental
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # required to access tags
+          submodules: "true"
+
+      - name: Build & run devcontainer image
+        # this always builds a new image from scratch rather than from the build-devcontainer-image workflow output
+        # so that we pick up the latest versions of everything
+        # NB: if you update this also update live-validation.yml and create-release-official.yaml
```

Comment:
There are similar comments in `create-release.yml`, `live-validation.yml` and `pre-release-tests.yaml` - all four files should reference each other so that someone making a change is clued-in to the need to keep all four in sync.

Reply:
Updated all references and opened https://github.com/Azure/azure-service-operator/issues/3712